client/src/components/root/ContainerDefeat.vue

```diff
@@ -42,45 +49,42 @@
 </template>
 
 <script lang="ts">
-import { Vue, Component, InjectReactive, Inject } from 'vue-property-decorator';
-import { GameRequestAPI } from '@port-of-mars/client/api/game/request';
-import {MarsLogData, ROLES} from '@port-of-mars/shared/types';
+  import { Component, Inject, Vue } from 'vue-property-decorator';
+  import { GameRequestAPI } from '@port-of-mars/client/api/game/request';
+  import { MarsLogData } from '@port-of-mars/shared/types';
 
-@Component({})
-export default class ContainerDefeat extends Vue {
-  @Inject() readonly api!: GameRequestAPI;
+  @Component({})
+  export default class ContainerDefeat extends Vue {
+    @Inject() readonly api!: GameRequestAPI;
 
-  // private handleRestart() {
-  //   this.api.resetGame();
-  // }
+    // private handleRestart() {
+    //   this.api.resetGame();
+    // }
+    get logs() {
+      return this.$tstore.getters.logs.reverse();
+    }
 
-  get marsLogEvents(){
-    console.log(this.$store.getters.marsLogEvents);
-    return this.$store.getters.logs.reverse();
-  }
+    logTime(timestamp: number) {
+      return new Date(timestamp).toLocaleTimeString();
+    }
 
-  marsLogTime(timestamp: number) {
-    return new Date(timestamp).toLocaleTimeString();
-  }
+    logColor(log: MarsLogData) {
+      if (log.category == 'SYSTEM HEALTH: DECREASE') return {backgroundColor: 'var(--marslog-red)'};
```

Comment:
it might be good to refactor this to a dictionary that maps the `log.category` to the color. Then you can boil this down to `return { backgroundColor: logCategoryColor[log.category] }`